### PR TITLE
 Added additional properties to the existing volume creation

### PR DESCRIPTION
**Pull request by chauberahul1993**

### CODE REVIEW

File: svc-systems/systems/storage.go [hunk 1]
```diff
@@ -127,7 +127,9 @@ func (e *ExternalInterface) CreateVolume(req *systemsproto.VolumeRequest) respon
 		response := common.GeneralError(http.StatusBadRequest, response.PropertyUnknown, errorMessage, []interface{}{invalidProperties}, nil)
 		return response
 	}
-
+	fmt.Printf("Volume Data is %+v ", volume)
```
Comment (hemkapoor) [inline svc-systems/systems/storage.go:130]: do we need this print statement? or can we use loggers

File: svc-systems/systems/storage.go [hunk 2]
```diff
@@ -405,7 +459,7 @@ func (e *ExternalInterface) DeleteVolume(req *systemsproto.VolumeRequest) respon
 		}
 
 	}
-
+	fmt.Printf("System Request Body %+v \n ", req.RequestBody)
```
Comment (hemkapoor) [inline svc-systems/systems/storage.go:462]: can we use loggers instead?

File: svc-systems/systems/storage.go (hunk 1 above)
Comment (chauberahul1993) [reply svc-systems/systems/storage.go]: done

File: svc-systems/systems/storage.go (hunk 2 above)
Comment (chauberahul1993) [reply svc-systems/systems/storage.go]: done

**Review by jeevan-kamkar — APPROVED** [review]

**Review by lpooja — APPROVED** [review]

**Review by shivacharanms — APPROVED** [review]

**Pull request merged**